src/Compilers/VisualBasic/Portable/Parser/ParseExpression.vb

```diff
@@ -9,7 +9,7 @@ Imports InternalSyntaxFactory = Microsoft.CodeAnalysis.VisualBasic.Syntax.Intern
 
 Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 
-    Friend Partial Class Parser
+    Partial Friend Class Parser
```

Comment:
Please revert

Reply:
@jaredpar  I think this is caused by an something in the IDE, possible pretty lister? Or Code Style enforcement? 
Writing `Friend Partial` is converted to `Partial Friend` automatically.